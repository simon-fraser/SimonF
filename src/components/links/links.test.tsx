import { render } from '@testing-library/react';
import { test, describe, expect } from 'vitest';
import React from 'react';
import Links from './links'

describe('components/icon0link', () => {
  test('will render correctly',() => {
    const {asFragment} = render(<Links />)

    expect(asFragment()).toMatchSnapshot();
  })
})