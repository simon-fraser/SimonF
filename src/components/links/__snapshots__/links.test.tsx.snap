// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`components/icon0link > will render correctly 1`] = `
<DocumentFragment>
  <aside
    class="absolute flex-col flex gap-2 h-screen justify-center left-2 z-10"
  >
    <a
      class="bg-black block duration-200 hover:-translate-y-0.5 hover:bg-gray-800 hover:shadow-xl p-1.5 rounded shadow transition"
      href="https://www.linkedin.com/in/frasersimon/"
      target="_blank"
    >
      <svg
        fill="none"
        height="24"
        stroke="white"
        stroke-linecap="round"
        stroke-linejoin="round"
        stroke-width="2"
        viewBox="0 0 24 24"
        width="24"
        xmlns="http://www.w3.org/2000/svg"
      >
        <path
          d="M16 8a6 6 0 0 1 6 6v7h-4v-7a2 2 0 0 0-2-2 2 2 0 0 0-2 2v7h-4v-7a6 6 0 0 1 6-6z"
        />
        <rect
          height="12"
          width="4"
          x="2"
          y="9"
        />
        <circle
          cx="4"
          cy="4"
          r="2"
        />
      </svg>
    </a>
    <a
      class="bg-black block duration-200 hover:-translate-y-0.5 hover:bg-gray-800 hover:shadow-xl p-1.5 rounded shadow transition"
      href="https://github.com/simon-fraser"
      target="_blank"
    >
      <svg
        fill="none"
        height="24"
        stroke="white"
        stroke-linecap="round"
        stroke-linejoin="round"
        stroke-width="2"
        viewBox="0 0 24 24"
        width="24"
        xmlns="http://www.w3.org/2000/svg"
      >
        <path
          d="M9 19c-5 1.5-5-2.5-7-3m14 6v-3.87a3.37 3.37 0 0 0-.94-2.61c3.14-.35 6.44-1.54 6.44-7A5.44 5.44 0 0 0 20 4.77 5.07 5.07 0 0 0 19.91 1S18.73.65 16 2.48a13.38 13.38 0 0 0-7 0C6.27.65 5.09 1 5.09 1A5.07 5.07 0 0 0 5 4.77a5.44 5.44 0 0 0-1.5 3.78c0 5.42 3.3 6.61 6.44 7A3.37 3.37 0 0 0 9 18.13V22"
        />
      </svg>
    </a>
  </aside>
</DocumentFragment>
`;
